docs/getting-started/ZE-system-reqs.md

```diff
@@ -1,13 +1,15 @@
-# Zowe Explorer System Requirements
+# Zowe Explorer system requirements
 
 Before installing Zowe Explorer, make sure that you meet the following requirements.
 
+:::info Required roles: systems administrator, devops architect
+:::
+
 ## Client side requirements
 
 ### Operating systems
 
-
-- MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey)
+- MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey), 13 (Ventura), 14 (Sonoma) **are all these correct?**
```

Comment:
@adam-wolfe please respond

Reply:
Somebody on the squad might know different, but it looks like 10.15 and 11 are no longer actively supported by Apple and VS Code only claims support for actively maintained versions of MacOS. So, we could consider removing 10.15 and 11.

Reply:
updated per ZE squad discussion